include/zephyr/drivers/sensor.h

```diff
@@ -1124,6 +1124,29 @@ static inline int sensor_value_from_float(struct sensor_value *val, float inp)
 	return 0;
 }
 
+/**
+ * @brief Helper function for converting int32_t to struct sensor_value.
+ *
+ * @param val A pointer to a sensor_value struct.
+ * @param inp The converted value.
```

Comment:
Looks like this comment needs updating, also for all the other `_to_` functions. Probably an old copy-paste error that never was caught.


```suggestion
 * @param inp The value to convert.
```